src/test/java/org/folio/test/support/ApiTests.java

```diff
@@ -148,7 +150,7 @@ protected static TenantAttributes getTenantAttributes() {
 
     return new TenantAttributes()
       .withModuleFrom(MODULE_NAME + "-14.2.4")
-      .withModuleTo(MODULE_NAME + "-" + getModuleVersion())
+      .withModuleTo(PomUtils.getModuleId())
```

Comment:
Unlike `getModuleId`, method `getModuleVersion()` returns current module version with `*-SNAPSHOT` suffix, which results in migration scripts running regardless of `fromModuleVersion` specified in the schema